components/src/main/res/drawable/components_round_circle.xml

```diff
@@ -0,0 +1,10 @@
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can we name this to reflect that it is used for the `MenuPanelRow`? Same for the other one too.